pyproject.toml

```diff
@@ -36,6 +36,7 @@ langchain-fireworks = { path = "libs/partners/fireworks", develop = true }
 langchain-google-vertexai = { git = "https://github.com/langchain-ai/langchain-google.git", subdirectory = "libs/vertexai" }
 langchain-groq = { path = "libs/partners/groq", develop = true }
 langchain-mistralai = { path = "libs/partners/mistralai", develop = true }
+langchain-neo4j = { path = "libs/partners/neo4j", develop = true }
```

Comment:
```suggestion
```

I don't think this addition is needed here. This pyproject.toml controls monorepo dependencies, which are primarily used for building and testing documentation.